scripts/reset_test_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Make sure to give this script execution permission (chmod +x reset_test_db.sh)
+
+# Check if the script is run as root (sudo)
+if [ "$EUID" -ne 0 ]; then
+  echo "Please run this script with sudo."
+  exit 1
+fi
```

Comment:
this isn't a requirement if your user is in the docker group (for example, I do not use sudo to execute docker commands on any of my dev machines)

Reply:
Fixed :))